bazel/external_crates.bzl

```diff
@@ -468,6 +468,9 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
             "group": crate.spec(
                 version = "^0.13",
             ),
+            "ic-sha3": crate.spec(
+                version = "^1.0.0",
+            ),
```

Comment:
since the crate lives in this repo, I'd rather depend on the code in the repo. Can you explain why we take a detour through crates.io?

Reply:
> I'd rather depend on the code in the repo

Any particular reason why?

> Can you explain why we take a detour through crates.io?

1. As per PR description, a next PR will publish `ic-ethereum-types` so that a dependency on the version on crates.io is a requirement
2. I also put the ckETH minter to depend on the version on crates.io, which while not strictly necessary, ensures that if we need anything new (unlikely) from that crate, we have an incentive of publishing a new version.

Reply:
> Any particular reason why?

Yep two reasons:
* Ensure the code within the repo is consistent
* Avoid building the crate twice

> a next PR will publish ic-ethereum-types

how about the `cketh` minter and `bls12_381`? will those also be published and/or use `ic-sha3` from crates.io?

Reply:
1. I discussed with @fspreiss about it yesterday and he (as a crypto owner) prefers to use the one in `//packages/` for `bls12_381`.
2. There is no plan to publish the ckETH minter on crates.io. If you feel strongly about it, I can let the minter depend on the one in `packages`, would that be preferable?

Reply:
gotcha, thanks for all the context. Then let's go ahead with this and maybe revisit in the future!